packages/terra-application-navigation/src/utility-menu/_UtilityMenu.jsx

```diff
@@ -78,6 +90,7 @@ const UtilityMenu = ({
 
   if (onSelectHelp) {
     menuItems.push({
+      id: utilityItemId(id, 'TerraHelp'),
```

Comment:
Would we want to make a special function that generates this rather than inputting 'TerraHelp'?

Reply:
I lean towards making a special function, but it's probably fine either way.

Reply:
Added special function to generate the help, settings, and logout ids